### DISCUSSION TITLE
what to do with obviously April's Fools "informational" submissions ?

### DISCUSSION THREAD

**Discussion by mogando668** [ietf-tools/datatracker#5536] [Help / Questions] [answered]
https://www.rfc-editor.org/rfc/rfc9402.txt

( note the date of its publishing, and the whimsical nature of its contents)

are these allowed  to remain with an RFC number ?

**Comment by cabo** [comment, marked as answer]
https://en.wikipedia.org/wiki/April_Fools%27_Day_Request_for_Comments

**Comment by KAMORU9** [comment]
Let everything right out now front of the A-Z property tax administration management thanks

**Reply by KAMORU9** [reply]
I won't morning Documents professional Administration LLC CEO Attracti
